src/dvclive/studio.py

```diff
@@ -71,15 +60,18 @@ def get_studio_updates(live):
     metrics_file = _adapt_path(live, metrics_file)
     metrics = {metrics_file: {"data": metrics}}
 
-    plots = {
-        _adapt_path(live, name): _adapt_plot_datapoints(live, plot)
-        for name, plot in plots.items()
-    }
-    plots = {k: {"data": v} for k, v in plots.items()}
+    plots_to_send = {}
+    for name, plot in plots.items():
+        path = _adapt_path(live, name)
+        nb_points_already_sent = live._nb_points_sent_to_studio.get(path, 0)
+        live._nb_points_sent_to_studio[path] = len(plot)
```

Comment:
Should we do this later after the call succeeds? Will this skip points if a call fails?